### PR TITLE
Finished building insights backend. Resolves KD-6 and KD-8

### PR DESCRIPTION
**Pull request by dwight-kaizenfox**
Finished code changes required for tickets: KD-6 and KD-8

### CODE REVIEW

File: page.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import Image from "next/image";
+
+export default function Home() {
+  return (
+    <main className="flex min-h-screen flex-col items-center justify-between p-24">
+      <div className="z-10 w-full max-w-5xl items-center justify-between font-mono text-sm lg:flex">
+        <p className="fixed left-0 top-0 flex w-full justify-center border-b border-gray-300 bg-gradient-to-b from-zinc-200 pb-6 pt-8 backdrop-blur-2xl dark:border-neutral-800 dark:bg-zinc-800/30 dark:from-inherit lg:static lg:w-auto  lg:rounded-xl lg:border lg:bg-gray-200 lg:p-4 lg:dark:bg-zinc-800/30">
+          Get started by editing&nbsp;
+          <code className="font-mono font-bold">src/app/page.tsx</code>
+        </p>
```
Comment (harshit040591) [inline page.tsx:10]: This section is unnecessary

File: page.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        </div>
+      </div>
+
+      <div className="relative flex place-items-center before:absolute before:h-[300px] before:w-[480px] before:-translate-x-1/2 before:rounded-full before:bg-gradient-radial before:from-white before:to-transparent before:blur-2xl before:content-[''] after:absolute after:-z-20 after:h-[180px] after:w-[240px] after:translate-x-1/3 after:bg-gradient-conic after:from-sky-200 after:via-blue-200 after:blur-2xl after:content-[''] before:dark:bg-gradient-to-br before:dark:from-transparent before:dark:to-blue-700 before:dark:opacity-10 after:dark:from-sky-900 after:dark:via-[#0141ff] after:dark:opacity-40 before:lg:h-[360px] z-[-1]">
+        <Image className="relative dark:drop-shadow-[0_0_0.3rem_#ffffff70] dark:invert" src="/next.svg" alt="Next.js Logo" width={180} height={37} priority />
+      </div>
+
+      <div className="mb-32 grid text-center lg:mb-0 lg:grid-cols-4 lg:text-left">
+        <a
+          href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
```
Comment (harshit040591) [inline page.tsx:29]: Please resolve this compiler warning

**Comment by harshit040591** [comment]
I've added 2 comments. Please fix those issues

**Review by harshit040591 — CHANGES_REQUESTED** [review]

File: page.tsx (hunk 1 above)
Comment (dwight-kaizenfox) [reply page.tsx:10]: This is a placeholder to the new page

**Review by harshit040591 — APPROVED** [review]

**Review by john-kaizenfox — APPROVED** [review]

**Pull request merged**